roles/contiv_network/tasks/main.yml

```diff
@@ -68,3 +68,6 @@
   args:
     chdir: /usr/bin/
     creates: contivctl
+
+- include: aci_tasks.yml
+  when: (run_as == "master") and (aci_mode is defined) and (aci_mode|bool)
```

Comment:
@DivyaVavili good idea, this can be a contiv_network level role variable. I like that actually as it is specific to the network role and not used otherwise.